doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc

```diff
@@ -1,15 +1,28 @@
 [id='creating-self-contained-image-proc_{context}']
 = Creating a self-contained {PRODUCT} Spring Boot JAR file
 
-You can create a self-contained {PRODUCT} Spring Boot JAR file that contains multiple versions of the same KJAR file, including modules. These are files that have the same `artifactID` and `groupID` attribute values, but have different `version` values. The KJAR files are separated from the `BOOT-INF/lib` JAR files to avoid class loader collisions. Each KJAR classpath container file is isolated from other KJAR classpath container files and does not rely on the Spring Boot class loader.
+You can create a self-contained {PRODUCT} Spring Boot JAR file that contains a complete service, including the {KIE_SERVER} and one or several KJAR files. This JAR file does not depend on loading any KJAR files at runtime.
```

Comment:
It's just {KIE_SERVER} when it's a noun (no the). If it is an adjective then use an article, For example,  'the {KIE_SERVER} instance' is correct.